### PR TITLE
Fixes grammar/typos/markdown

### PR DESCRIPTION
**Pull request by nikoandpiko**
### Prerequisites

- [ √ ] I have written a descriptive pull-request title
- [ √ ] I have verified that there are no overlapping [pull-requests](https://github.com/ImageMagick/ImageMagick/pulls) open
- [ √ ] I have verified that I am following the existing coding patterns and practices as demonstrated in the repository.

### Description
Fixes various typos, grammar, and formatting across several root files.

### CODE REVIEW

File: NEWS.txt [hunk 1]
```diff
@@ -15,9 +15,9 @@ ImageMagick release, we perform a comprehensive security assessment that
 includes memory error, thread data race detection, and continuous fuzzing
 to detect and prevent security vulnerabilities.
 
-As an analog to linear (RGB) and non-linear (sRGB) color colorspaces, as
```
Comment (dlemstra) [inline NEWS.txt:18]: We use `non-linear` every instead so can you revert this? I found [this](https://english.stackexchange.com/a/172298) reference that both are allowed:
> The Oxford Dictionary (which I personally consider the reference for the English language) lists "non-linear" as the correct British spelling and no alternative is given. But if you switch to the US English dictionary, the only entry is "nonlinear".

File: NOTICE [hunk 2]
```diff
@@ -8,14 +8,14 @@ to making software imaging solutions freely available.
 You may not use this file except in compliance with the License. You may obtain
 a copy of the License at
 
-  https://imagemagick.org/script/license.php
+  <https://imagemagick.org/script/license.php>
```
Comment (dlemstra) [inline NOTICE:11]: Can you revert this here and in other spots? This is not a grammatical change?

File: README.md [hunk 3]
```diff
@@ -10,16 +10,16 @@
 
 Use [ImageMagick®](https://imagemagick.org/) to create, edit, compose, or convert digital images. It can read and write images in a variety of formats (over 200) including PNG, JPEG, GIF, WebP, HEIC, SVG, PDF, DPX, EXR and TIFF. ImageMagick can resize, flip, mirror, rotate, distort, shear and transform images, adjust image colors, apply various special effects, or draw text, lines, polygons, ellipses and Bézier curves.
 
-#### What is ImageMagick?
+## What is ImageMagick?
```
Comment (dlemstra) [inline README.md:13]: Can you please revert these changes?

File: README.md [hunk 4]
```diff
@@ -64,11 +64,11 @@ Here are just a few [examples](https://imagemagick.org/script/examples.php) of w
 
 [Examples of ImageMagick Usage](https://legacy.imagemagick.org/Usage/), shows how to use ImageMagick from the command-line to accomplish any of these tasks and much more. Also, see [Fred's ImageMagick Scripts](http://www.fmwconcepts.com/imagemagick/): a plethora of command-line scripts that perform geometric transforms, blurs, sharpens, edging, noise removal, and color manipulations. With [Magick.NET](https://github.com/dlemstra/Magick.NET), use ImageMagick without having to install ImageMagick on your server or desktop.
 
-#### News
+## News
 
-ImageMagick best practices **strongly** encourages you to configure a [security policy](https://imagemagick.org/script/security-policy.php) that suits your local environment.
+ImageMagick best practices **strongly** encourage you to configure a [security policy](https://imagemagick.org/script/security-policy.php) that suits your local environment.
```
Comment (dlemstra) [inline README.md:69]: This should be `encourages`?

**Review by dlemstra — COMMENTED** [review]
Thanks for helping us by fixing these grammatical errors. Left some comments about things that we would like reverted. If you rather have us do that ourselves then please let me know.

**Comment by nikoandpiko** [comment]
@dlemstra Thank you for the feedback. Addressed everything you mentioned and found an additional two files where some typos remained.

**Review by dlemstra — APPROVED** [review]
Thanks for helping us out!
